tools/chaincode-integration/src/step-definitions/channel/channel.ts

```diff
@@ -39,14 +39,19 @@ export class Channel {
         await this.workspace.infrastructureProvider.deployCC(this.workspace, channelName, chaincodeName);
     }
 
-    @given(/Chaincode ['"](.*)['"] has been started in server mode$/, '', TIMEOUT)
-    public async startCCAAS(chaincodeName: string): Promise<void> {
-        await this.workspace.infrastructureProvider.startCCAAS(this.workspace, chaincodeName);
-    }
+    // TODO delete me?
+    // @given(/Chaincode ['"](.*)['"] has been started in server mode$/, '', TIMEOUT)
+    // public async startCCAAS(chaincodeName: string): Promise<void> {
+    //     await this.workspace.infrastructureProvider.startCCAAS(this.workspace, chaincodeName);
+    // }
 
     @given(/All peers on channel ['"](.*)['"] have deployed the chaincode-as-a-service ['"](.*)['"]$/, '', TIMEOUT)
     public async deployCCAAS(channelName: string, chaincodeName: string): Promise<void> {
-        await this.workspace.infrastructureProvider.deployCC(this.workspace, channelName, `${chaincodeName}-ccaas`);
+        // Removed "-ccaas" suffix because it causes problems with finding the chaincode config
+        //   TypeError: Cannot read properties of undefined (reading 'path')
+        //   at TestNetworkProvider.deployCCAAS (/Users/jamest/Code/fabric-test/tools/chaincode-integration/src/infrastructure/testNetworkProvider.ts:89:96)
+        // TODO are there any implications? i.e. same chaincode name as a service and as normal in same test run?
```

Comment:
won't there be a collision in the peer if there's a chaincode of the same name?

Reply:
I was hoping that different features would start with a clean slate, but apparently not